front/tests/015-train-timetable.spec.ts

```diff
@@ -88,33 +88,33 @@ test.describe('Verify train schedule elements and filters', () => {
   /** *************** Test 1 **************** */
   test('Loading trains and verifying simulation result for trainSchedule', async () => {
```

Comment:
Same for l.101 (guess we could change also some `train` in l.102 as well)
```suggestion
  test('Loading timetable items and verifying simulation result for trainSchedule', async () => {
```